src/server/worker/api_server.go

```diff
@@ -1374,8 +1379,11 @@ func (a *APIServer) mergeDatums(jobCtx context.Context, pachClient *client.APICl
 }
 
 func (a *APIServer) getChunk(ctx context.Context, id int64, address string, failed bool) error {
-	// If this worker processed the chunk, then it is already in the chunk cache
+	// If this worker processed the chunk, make sure it is already in the chunk cache
```

Comment:
Maybe this should be `If we think this worker processed this chunk...`. Also, I would be curious to know if the actual problem is recycled IPs.

Reply:
I think this situation could arise if a job errored and resumed in the `MERGING` state, even without recycling workers or IP addresses.  I haven't done a full audit of our error behavior to see exactly what's possible, though.